docs/troubleshoot/cli/known-cli.md

```diff
@@ -116,28 +129,50 @@ issue the following command:
 npm install npm@5.3.0 -g
 ```
 
-## Node.js commands do not respond as expected
+## Paths converting in Git Bash
 
-**Valid on Windows or Linux**
+**Valid on Windows**
 
 **Symptom:**
 
-You attempt to issue node.js commands and you do not receive the expected  output.
+When issuing a command with a relative directory path using forward slashes, Git Bash converts the path into an absolute path with a root directory.
 
-**Solution:**
+**Solutions:**
 
-There might be a program that is named *node* on your path. The Node.js installer automatically adds a program that is named *node* to your path. When there are pre-existing programs that are named *node* on your computer, the program that appears first in the path is used. To correct this behavior, change the order of the programs in the path so that Node.js appears first.
+- Replace the single slash in front of a path with double forward slashes (//). This stops Git Bash from remapping the path.
 
-## Installation fails on Oracle Linux 6
+  Git Bash converts the path entered in the following command:
 
-**Valid on Oracle Linux 6**
+  ```
+  zowe zos-files upload dir-to-uss local_dir /a/ibmuser/my_dir
+  ```
+  To avoid conversion, the path should be edited as in the following example:
+  ```
+  zowe zos-files upload dir-to-uss local_dir //a/ibmuser/my_dir
+  ```
```

Comment:
Maybe we should add an example of how the path gets converted and what type of error the user may see.

In this case, the path gets converted to `A:/ibmuser/my_dir` and the user may see a "No such file or directory" error.

Reply:
per our discussion, will wait on you to provide an example